src/main.js

```diff
@@ -2,6 +2,40 @@
 const THREE = require('three'); // older modules are imported like this. You shouldn't have to worry about this much
 import Framework from './framework'
 
+
+
+
+var parameters = {
+    octaves: 2.0,
+    persistence: 4.0
+}
+
+var noiseMaterial = new THREE.ShaderMaterial({
+     uniforms: {
+      // image: { // Check the Three.JS documentation for the different allows types and values
+      //   type: "t",
+      //   value: THREE.ImageUtils.loadTexture('./explosion.png')
+      // },
+      time: {
+        type: "f",
+        value: 1.0
+      },
+      octaves: {
+        type: "f",
+        value: 3.0
+      },
+      persistence: {
+        type: "f",
+        value: 4.0
```

Comment:
This value should probably be bounded from 0 to 1. Greater than 1 and your function grows out of control as you increase octaves.